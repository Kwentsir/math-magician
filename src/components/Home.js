import React from 'react';

const Home = () => (
  <div className="home">
    <h3>Welcome to our page!</h3>
    <p>
      This is a simple basic calculator with memory functions similar to a small
      handheld calculator. Use this basic calculator online for math with
      addition, subtraction, division and multiplication.
    </p>
  </div>
);

export default Home;
