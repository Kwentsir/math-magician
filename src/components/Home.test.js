import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('Correctly renders a component', () => {
  it('renders home component', () => {
    const component = renderer.create(<Home />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
