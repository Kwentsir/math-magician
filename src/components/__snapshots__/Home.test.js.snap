// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders a component renders home component 1`] = `
<div
  className="home"
>
  <h3>
    Welcome to our page!
  </h3>
  <p>
    This is a simple basic calculator with memory functions similar to a small handheld calculator. Use this basic calculator online for math with addition, subtraction, division and multiplication.
  </p>
</div>
`;
